File: OPENAPI-REST-API/swagger-client/spring/src/main/java/io/swagger/model/VpsRow.java
package io.swagger.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import org.springframework.validation.annotation.Validated;
import org.openapitools.jackson.nullable.JsonNullable;
import io.swagger.configuration.NotUndefined;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonSetter;
import com.fasterxml.jackson.annotation.Nulls;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * A result row from the &#x60;Vps&#x60; &#x60;GET&#x60; request.
 */
@Schema(description = "A result row from the `Vps` `GET` request.")
@Validated
@NotUndefined



public class VpsRow   {
  @JsonProperty("vps_id")

  private String vpsId = null;

  @JsonProperty("vps_name")

  private String vpsName = null;

  @JsonProperty("repeat_invoices_cost")

  private String repeatInvoicesCost = null;

  @JsonProperty("vps_hostname")

  private String vpsHostname = null;

  @JsonProperty("vps_ip")

  private String vpsIp = null;

  @JsonProperty("vps_status")

  private String vpsStatus = null;

  @JsonProperty("services_name")

  private String servicesName = null;

  @JsonProperty("vps_comment")

  private String vpsComment = null;


  public VpsRow vpsId(String vpsId) { 

    this.vpsId = vpsId;
    return this;
  }

  /**
   * The id of the vps.
   * @return vpsId
   **/
  
  @Schema(example = "100", required = true, description = "The id of the vps.")
  
  @NotNull
  public String getVpsId() {  
    return vpsId;
  }



  public void setVpsId(String vpsId) { 

    this.vpsId = vpsId;
  }

  public VpsRow vpsName(String vpsName) { 

    this.vpsName = vpsName;
    return this;
  }

  /**
   * The name of the vps.
   * @return vpsName
   **/
  
  @Schema(example = "Builder", required = true, description = "The name of the vps.")
  
  @NotNull
  public String getVpsName() {  
    return vpsName;
  }



  public void setVpsName(String vpsName) { 

    this.vpsName = vpsName;
  }

  public VpsRow repeatInvoicesCost(String repeatInvoicesCost) { 

    this.repeatInvoicesCost = repeatInvoicesCost;
    return this;
  }

  /**
   * The repeat invoices cost of the vps.
   * @return repeatInvoicesCost
   **/
  
  @Schema(example = "10.00", required = true, description = "The repeat invoices cost of the vps.")
  
  @NotNull
  public String getRepeatInvoicesCost() {  
    return repeatInvoicesCost;
  }



  public void setRepeatInvoicesCost(String repeatInvoicesCost) { 

    this.repeatInvoicesCost = repeatInvoicesCost;
  }

  public VpsRow vpsHostname(String vpsHostname) { 

    this.vpsHostname = vpsHostname;
    return this;
  }

  /**
   * The hostname of the vps.
   * @return vpsHostname
   **/
  
  @Schema(example = "vps100", required = true, description = "The hostname of the vps.")
  
  @NotNull
  public String getVpsHostname() {  
    return vpsHostname;
  }



  public void setVpsHostname(String vpsHostname) { 

    this.vpsHostname = vpsHostname;
  }

  public VpsRow vpsIp(String vpsIp) { 

    this.vpsIp = vpsIp;
    return this;
  }

  /**
   * The ip of the vps.
   * @return vpsIp
   **/
  
  @Schema(example = "64.20.46.220", required = true, description = "The ip of the vps.")
  
  @NotNull
  public String getVpsIp() {  
    return vpsIp;
  }



  public void setVpsIp(String vpsIp) { 

    this.vpsIp = vpsIp;
  }

  public VpsRow vpsStatus(String vpsStatus) { 

    this.vpsStatus = vpsStatus;
    return this;
  }

  /**
   * The status of the vps.
   * @return vpsStatus
   **/
  
  @Schema(example = "active", required = true, description = "The status of the vps.")
  
  @NotNull
  public String getVpsStatus() {  
    return vpsStatus;
  }



  public void setVpsStatus(String vpsStatus) { 

    this.vpsStatus = vpsStatus;
  }

  public VpsRow servicesName(String servicesName) { 

    this.servicesName = servicesName;
    return this;
  }

  /**
   * The services name of the vps.
   * @return servicesName
   **/
  
  @Schema(example = "KVM Windows VPS Slice", required = true, description = "The services name of the vps.")
  
  @NotNull
  public String getServicesName() {  
    return servicesName;
  }



  public void setServicesName(String servicesName) { 

    this.servicesName = servicesName;
  }

  public VpsRow vpsComment(String vpsComment) { 

    this.vpsComment = vpsComment;
    return this;
  }

  /**
   * The comment of the vps.
   * @return vpsComment
   **/
  
  @Schema(required = true, description = "The comment of the vps.")
  
  @NotNull
  public String getVpsComment() {  
    return vpsComment;
  }



  public void setVpsComment(String vpsComment) { 

    this.vpsComment = vpsComment;
  }

  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    VpsRow vpsRow = (VpsRow) o;
    return Objects.equals(this.vpsId, vpsRow.vpsId) &&
        Objects.equals(this.vpsName, vpsRow.vpsName) &&
        Objects.equals(this.repeatInvoicesCost, vpsRow.repeatInvoicesCost) &&
        Objects.equals(this.vpsHostname, vpsRow.vpsHostname) &&
        Objects.equals(this.vpsIp, vpsRow.vpsIp) &&
        Objects.equals(this.vpsStatus, vpsRow.vpsStatus) &&
        Objects.equals(this.servicesName, vpsRow.servicesName) &&
        Objects.equals(this.vpsComment, vpsRow.vpsComment);
  }

  @Override
  public int hashCode() {
    return Objects.hash(vpsId, vpsName, repeatInvoicesCost, vpsHostname, vpsIp, vpsStatus, servicesName, vpsComment);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class VpsRow {\n");
    
    sb.append("    vpsId: ").append(toIndentedString(vpsId)).append("\n");
    sb.append("    vpsName: ").append(toIndentedString(vpsName)).append("\n");
    sb.append("    repeatInvoicesCost: ").append(toIndentedString(repeatInvoicesCost)).append("\n");
    sb.append("    vpsHostname: ").append(toIndentedString(vpsHostname)).append("\n");
    sb.append("    vpsIp: ").append(toIndentedString(vpsIp)).append("\n");
    sb.append("    vpsStatus: ").append(toIndentedString(vpsStatus)).append("\n");
    sb.append("    servicesName: ").append(toIndentedString(servicesName)).append("\n");
    sb.append("    vpsComment: ").append(toIndentedString(vpsComment)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/openapi-client/cpp-tiny/lib/Models/VpsRow.cpp


#include "VpsRow.h"

using namespace Tiny;

VpsRow::VpsRow()
{
	vps_id = std::string();
	vps_name = std::string();
	repeat_invoices_cost = std::string();
	vps_hostname = std::string();
	vps_ip = std::string();
	vps_status = std::string();
	services_name = std::string();
	vps_comment = std::string();
}

VpsRow::VpsRow(std::string jsonString)
{
	this->fromJson(jsonString);
}

VpsRow::~VpsRow()
{

}

void
VpsRow::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);

    const char *vps_idKey = "vps_id";

    if(object.has_key(vps_idKey))
    {
        bourne::json value = object[vps_idKey];



        jsonToValue(&vps_id, value, "std::string");


    }

    const char *vps_nameKey = "vps_name";

    if(object.has_key(vps_nameKey))
    {
        bourne::json value = object[vps_nameKey];



        jsonToValue(&vps_name, value, "std::string");


    }

    const char *repeat_invoices_costKey = "repeat_invoices_cost";

    if(object.has_key(repeat_invoices_costKey))
    {
        bourne::json value = object[repeat_invoices_costKey];



        jsonToValue(&repeat_invoices_cost, value, "std::string");


    }

    const char *vps_hostnameKey = "vps_hostname";

    if(object.has_key(vps_hostnameKey))
    {
        bourne::json value = object[vps_hostnameKey];



        jsonToValue(&vps_hostname, value, "std::string");


    }

    const char *vps_ipKey = "vps_ip";

    if(object.has_key(vps_ipKey))
    {
        bourne::json value = object[vps_ipKey];



        jsonToValue(&vps_ip, value, "std::string");


    }

    const char *vps_statusKey = "vps_status";

    if(object.has_key(vps_statusKey))
    {
        bourne::json value = object[vps_statusKey];



        jsonToValue(&vps_status, value, "std::string");


    }

    const char *services_nameKey = "services_name";

    if(object.has_key(services_nameKey))
    {
        bourne::json value = object[services_nameKey];



        jsonToValue(&services_name, value, "std::string");


    }

    const char *vps_commentKey = "vps_comment";

    if(object.has_key(vps_commentKey))
    {
        bourne::json value = object[vps_commentKey];



        jsonToValue(&vps_comment, value, "std::string");


    }


}

bourne::json
VpsRow::toJson()
{
    bourne::json object = bourne::json::object();





    object["vps_id"] = getVpsId();






    object["vps_name"] = getVpsName();






    object["repeat_invoices_cost"] = getRepeatInvoicesCost();






    object["vps_hostname"] = getVpsHostname();






    object["vps_ip"] = getVpsIp();






    object["vps_status"] = getVpsStatus();






    object["services_name"] = getServicesName();






    object["vps_comment"] = getVpsComment();



    return object;

}

std::string
VpsRow::getVpsId()
{
	return vps_id;
}

void
VpsRow::setVpsId(std::string vps_id)
{
	this->vps_id = vps_id;
}

std::string
VpsRow::getVpsName()
{
	return vps_name;
}

void
VpsRow::setVpsName(std::string vps_name)
{
	this->vps_name = vps_name;
}

std::string
VpsRow::getRepeatInvoicesCost()
{
	return repeat_invoices_cost;
}

void
VpsRow::setRepeatInvoicesCost(std::string repeat_invoices_cost)
{
	this->repeat_invoices_cost = repeat_invoices_cost;
}

std::string
VpsRow::getVpsHostname()
{
	return vps_hostname;
}

void
VpsRow::setVpsHostname(std::string vps_hostname)
{
	this->vps_hostname = vps_hostname;
}

std::string
VpsRow::getVpsIp()
{
	return vps_ip;
}

void
VpsRow::setVpsIp(std::string vps_ip)
{
	this->vps_ip = vps_ip;
}

std::string
VpsRow::getVpsStatus()
{
	return vps_status;
}

void
VpsRow::setVpsStatus(std::string vps_status)
{
	this->vps_status = vps_status;
}

std::string
VpsRow::getServicesName()
{
	return services_name;
}

void
VpsRow::setServicesName(std::string services_name)
{
	this->services_name = services_name;
}

std::string
VpsRow::getVpsComment()
{
	return vps_comment;
}

void
VpsRow::setVpsComment(std::string vps_comment)
{
	this->vps_comment = vps_comment;
}



